pages/initial-setup/index.html

```diff
@@ -1,12 +1,11 @@
 ---
 layout: default
 title: Initial Setup &amp; Deployment
-excerpt: We write the Terraform code for you, apply it to your AWS accounts, and push it to your git repos.
+excerpt: We define your entire infrastructure as code and deploy it into your AWS accounts in about 1 day.
 permalink: /initial-setup/
 slug: subscriptions
 footer_heading: Talk with our team of DevOps experts now!
-redirect_from:
-  - /reference-architecture/
```

Comment:
Isn't this the new RefArch page?

Reply:
I made it the Usage Patterns Library page